sc/src/main/java/uk/ac/ebi/atlas/tsne/TSnePoint.java

```diff
@@ -7,14 +7,14 @@
 import javax.annotation.Nullable;
 
 @AutoValue
-public abstract class TSnePoint  implements Comparable<TSnePoint> {
+public abstract class   TSnePoint  implements Comparable<TSnePoint> {
```

Comment:
This is confusing, it's not clear from the class why it is `Comparable`, and the definition of comparing them by name doesn't correspond to any innate property of TSnePoints. I figured out that it's so you can put these in TreeMaps but there's no domain specific reason to do this.

Reply:
👍

Reply:
I’m a bit unsure if the comparator should be in `TSnePoint` or in `TSnePlotCollator`. I went with the first option to reuse it in `RandomTSnePlot`, but when this goes away we can move it to the collator.